app/javascript/custom/pagination.js

```diff
@@ -0,0 +1,13 @@
+jQuery(function() {
+if ($('.pagination').length) {
+  $(window).scroll(function() {
```

Comment:
where r u calling this $(window).scroll() ?

Reply:
I have added an eventListener to detect the scrolling event.